src/torchaudio/__init__.py

```diff
@@ -41,6 +39,182 @@
     pass
 
 
+def load(
+    uri: Union[BinaryIO, str, os.PathLike],
+    frame_offset: int = 0,
+    num_frames: int = -1,
+    normalize: bool = True,
+    channels_first: bool = True,
+    format: Optional[str] = None,
+    buffer_size: int = 4096,
+    backend: Optional[str] = None,
+) -> Tuple[torch.Tensor, int]:
+    """Load audio data from source using TorchCodec's AudioDecoder.
+
+    .. note::
+
+        This function supports the same API as :func:`~torchaudio.load`, and
```

Comment:
this function *is* torchaudio.load(). Here and for `save()`  please make sure to update the docstrings so that they still make sense.

Similarly the paragraph below should be updated, and possibly other parts in the docstring.